src/pages/ServiceDetails/ServiceInfo.tsx

```diff
@@ -74,6 +74,21 @@ class ServiceInfo extends React.Component<ServiceDetails, ServiceInfoState> {
     return validationChecks;
   }
 
+  differentSources(): string[] {
+    const dependencies = this.props.serviceDetails.dependencies || {};
+
+    let differentDependencies: string[] = [];
+    Object.keys(dependencies).forEach(key => {
+      dependencies[key].forEach((dependency: SourceWorkload) => {
+        if (!differentDependencies.includes(dependency.name)) {
+          differentDependencies.push(dependency.name);
+        }
+      });
+    });
+
+    return differentDependencies;
```

Comment:
Why does this function return an array and not a number? the return array content is not used later, only the length (aka the number :)) is used.